gobblin-modules/gobblin-compliance/src/main/java/gobblin/compliance/ComplianceConfigurationKeys.java

```diff
@@ -33,6 +33,16 @@
   public static final String PARTITION_NAME = COMPLIANCE_PREFIX + ".partition.name";
   public static final int TIME_STAMP_LENGTH = 13;
   public static final String DBNAME_SEPARATOR = "__";
+  public static final String NUM_ROWS = "numRows";
+  public static final String RAW_DATA_SIZE = "rawDataSize";
+  public static final String TOTAL_DATA_SIZE = "totalDataSize";
+  public static final String WORKUNIT_RECORDSREAD = COMPLIANCE_PREFIX + ".workunit.recordsRead";
+  public static final String WORKUNIT_RECORDSWRITTEN = COMPLIANCE_PREFIX + ".workunit.recordsWritten";
+  public static final String WORKUNIT_BYTESREAD = COMPLIANCE_PREFIX + ".workunit.bytesRead";
+  public static final String WORKUNIT_BYTESWRITTEN = COMPLIANCE_PREFIX + ".workunit.bytesWritten";
+  public static final String EXECUTION_NUMBER = COMPLIANCE_PREFIX + ".workunit.executionNumberInCurrentCycle";
+  public static final String TOTAL_EXECUTIONS_IN_CYCLE = COMPLIANCE_PREFIX + ".workunit.totalExecutionsInCycle";
```

Comment:
Convention is to avoid terms like 'in' and 'of' in property names, if possible. Maybe totalExecutions is sufficient?